release-notes/8.0/preview/api-diff/rc1/Microsoft.NETCore.App/8.0-rc1_System.Numerics.md

```diff
@@ -0,0 +1,15 @@
+# System.Numerics
```

Comment:
System.Numerics LGTM.

I agree with Stephen's input that ideally we track `[RequiresLocationAttribute] ref` as `ref readonly`, which is the actual language syntax here.